rio/src/RIO/Prelude/Logger.hs

```diff
@@ -459,11 +466,42 @@ data LogOptions = LogOptions
   , logTerminal :: !Bool
   , logUseTime :: !Bool
   , logUseColor :: !Bool
+  , logColors :: !LogColors
   , logUseLoc :: !Bool
   , logFormat :: !(Utf8Builder -> Utf8Builder)
   , logSend :: !(Builder -> IO ())
   }
 
+-- | ANSI color codes for use in the configuration of the creation of a
+-- 'LogFunc'.
+--
+-- @since 0.1.18.0
+data LogColors = LogColors
```

Comment:
It seems surprising to me to export the constructors here, not export the `defaultLogColors` value, but yes export the setters. I'd lean towards not exporting the constructor and fields.

Reply:
I have revised the pull request. Rather than export `LogColors` and its three constructors, I export only three accessors.

Reply:
I think you'll also need to export `defaultLogColors`, otherwise it won't be possible to create a value to modify

Reply:
I think it is not correct to export identifier `defaultLogColors`, for the folowing reason: with one exception, the only way to create a `LogOptions` value (other than from an existing `LogOptions` value and using the `set...` functions) is to use `RIO.logOptionsHandle` (which is what returns the 'default' `LogOptions` value that is the starting point for others). (The exception is `RIO.logOptionsMemory` which is 'primarily intended for testing' - and also returns a 'default'/'empty' `LogOptions` value.) The 'defaults' are documented but they are 'behind the scenes'.